build-tools/create-packs/Directory.Build.targets

```diff
@@ -71,23 +72,23 @@
   </Target>
 
   <Target Name="BuildILLinkCustomStep">
-    <Exec Command="$(DotNetPreviewTool) build -p:Configuration=$(Configuration) &quot;$(XamarinAndroidSourcePath)src\Microsoft.Android.Sdk.ILLink\Microsoft.Android.Sdk.ILLink.csproj&quot;" />
+    <Exec Command="dotnet build -p:Configuration=$(Configuration) &quot;$(XamarinAndroidSourcePath)src\Microsoft.Android.Sdk.ILLink\Microsoft.Android.Sdk.ILLink.csproj&quot;" />
```

Comment:
Should we put a note, we could potentially build this project normally when we get #5806? Then `msbuild` on Mac could build `net5.0`, and we could put it in `Xamarin.Android.sln` like a normal project?

Reply:
I dropped a note on the other PR, we can potentially include these changes there 👍